### ISSUE TITLE
The error generated by the table entry having incorrect details is not read by NVDA on occurring

### ISSUE THREAD

**Issue by Phatal900** [nvaccess/nvda#19951]
### Brief summary

Hello we are seeing an issue where the error generated by the table entry having incorrect details is not read by NVDA on occurring.

The focus still moves to the problematic element when the user tries to Save & Continue.

Second, on Saving & Continuing when in error, the error is read out in NVDA.

### Steps to reproduce

1. Access the community portal forgot password page, https://mdccs--sit.sandbox.my.site.com/familyportal/s/login/ForgotPassword?language=en_US

2. Enter email neelutest20@yopmail.com and select the send button. 

3. Navigate to yopmail.com. Enter the email in the email field and select the arrow symbol to access to the inbox. 

4. Open the forgot password email. Select the Password Reset hyperlink.

5. In the inbox, find the email for the MFA code and enter that. Then select a new password. 

6. Once routed to the portal, go to the Continue your Scholarship Application section, and select the Continue Application button.  

7. Navigate through the application and select the Save & Continue button until you reach the Household Member Information screen. 

8. Select the Add Co-Applicant button. 

9. Enter all the required fields. For the email fields enter the below. 

test11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111@test.com

### Actual behavior

After selecting Save from the co-applicant popup modal, the error appears on-screen but it does not get read. 

### Expected behavior

After selecting Save from the co-applicant popup modal, the error appears on-screen and should be read.

### NVDA type

installed copy

### NVDA version

NVDA Version 2025.3.2 (2025.3.2.53588)

### Windows version

Windows 11 Enterprise23H2 Build 22631.6491

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have run the COM Registration Fixing Tool and the issue still occurs

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

To be able to reproduce this issue, we need a minimal code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/). Please avoid libraries and ensure this is a truly minimal sample with plain HTML and JS.

Kind Regards,
NV Access Software Developers

**Comment by seanbudd** [comment]
Closing as we haven't received the required information. Please provide it and we can re-open
